docs/Config.md

```diff
@@ -410,6 +410,15 @@ gui:
     'docs': '#11aaff' # use a light blue for branches beginning with 'docs/'
 ```
 
+## Custom Gone Branch Text
+
+You can customize the text for upstream gone branches
+
+```yaml
+gui:
+  goneBranchText: 'gone' # show 'gone' for upstream gone branches, otherwise use '' to hide gone status
```

Comment:
Could we add an up arrow before 'gone' to indicate that it is the upstream (tracked) branch which is gone? Also, can you confirm my understanding that this text will be printed in yellow (the default color for `ColoredBranchStatus`)?

Reply:
Up arrow is added. Yes, your understanding is correct the text will be printed in yello, but I think red that you made in #1803 makes more sense, so I updated that as well. Let me know if you want to incorporate the refactor and the test yourself or you want me to have a crack at it.

Reply:
I will open a PR for the refactor tomorrow morning EST. Thanks for the changes.

Reply:
PR opened: https://github.com/john-nguyen09/lazygit/pull/1

Reply:
Thanks @john-nguyen09